translator/conventions/opentelemetry_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conventions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDefinedAttributeValues(t *testing.T) {
+	attributes := make(map[string]string)
+	attributes[AttributeNetTransport] = TransportTypeTCP
```

Comment:
I am not sure I understand what this test verifies. Seems like it checks that the map works correctly. Is there a value in this test for this particular package?

Reply:
Since there are only constants in this package, there is nothing to test. But build fails without an test files so I added this dummy test to make the build pass.

Reply:
As a convention in this repo we create `empty-test.go` without any tests defined. That should be enough to pass the build. See e.g. `compression/empty-test.go`.

Reply:
I will fix